prime-router/src/testIntegration/kotlin/datatests/mappinginventory/nte_note/NTETests.kt

```diff
@@ -0,0 +1,46 @@
+package gov.cdc.prime.router.datatests.mappinginventory.nte_note
+
+import gov.cdc.prime.router.datatests.mappinginventory.translateAndCompareFHIRToHL7
+import gov.cdc.prime.router.datatests.mappinginventory.translateAndCompareHL7ToFHIR
+import org.junit.jupiter.api.Test
+
+// This test asserts against a sample HL7 that an NTE segment associated with a PID, OBR and OBX and verifies
+// that the no data is loss between each step.  The NTE segments associated with PID and OBX both have NTE.5 populated
+class HL7ToFhirToHL7 {
+    @Test
+    fun `can accurately map NTE segment to annotation resource`() {
```

Comment:
Hmm, In lieu (I had to google how to spell this word but it called to me) of tests that  verify each specific requirement, or row, of the mapping inventory for NTE, can we, or should we, add detailed description (perhaps in a comment) that details what transformations/mappings this test is actually testing? Otherwise, why not just come up with a naming scheme and a single tests that iterates over each fair of files and prints out which ones failed (not suggesting this is a good idea)?